### ISSUE TITLE
The profile picture is rotated 90° counterclockwise

### ISSUE THREAD

**Issue by Gh0stz0x** [deltachat/deltachat-android#1835]
- Operating System (Linux/Mac/Windows/iOS/Android): Android 10 - Lineage Os 17.1
- Delta Chat Version: 1.14.5 - nightly
- Expected behavior: When I select an image or take a photograph, I want it to stay where I took it.
- Actual behavior: When I select an image, it rotates 90° counterclockwise
- Steps to reproduce the problem: Go to settings, select the profile, change image by selecting it from the gallery or taking a photograph in portrait mode. The photograph will be rotated 90 ° counterclockwise. This bug was detected on Samsung and Xiaomi smartphones.
- Screenshots: 
![IMG_20210312_231019](https://user-images.githubusercontent.com/74795149/111003910-ee7a2680-837f-11eb-81e1-5f42bcbfe8cd.jpg)
- Logs:

**Comment by Hocuri** [comment]
Strange. I can't reproduce this on my Fairphone 3 w/ Android 10. 
And this happens both when you directly take a photo and when you select one from the gallery? 
Does it happen everytime or just sometimes? 
Did it only happen on LineageOS devices so far?
Does it also happen if the device is in landscape mode?

**Comment by r10s** [comment]
there is a related fix at https://github.com/deltachat/deltachat-core-rust/pull/2063, however, the fix is not release, but it is in the nigtlies (please double-check, that core 1.51.0-alpha.0 is used).

please note, that existing avatars are not rotated correctly after update, you can check that only with new avatars.

however, of course, may also be another issue :)

**Comment by Gh0stz0x** [comment]
@Hocuri It occurs either by taking a new photograph or by selecting a photograph from the gallery.
@r10s The user who reported it to me told me that he uses the nightly. I try to check on a Samsung that I have to reproduce the problem and see which core it uses. 
However, the bug exists on both Android and Lineage.

**Comment by Gh0stz0x** [comment]
I looked on my Samsung log: it uses 
1.51.0-alpha.0 core

**Comment by Gh0stz0x** [comment]
>Does it also happen if the device is in landscape >mode?

@Hocuri I'll give it a try and I'll tell you

**Comment by Gh0stz0x** [comment]
@Hocuri If I do the landscape photography the photograph is "straight".  Now I have asked the two users to test them too.

**Comment by adbenitez** [comment]
> there is a related fix at [deltachat/deltachat-core-rust#2063](https://github.com/deltachat/deltachat-core-rust/pull/2063), however, the fix is not release, but it is in the nigtlies (please double-check, that core 1.51.0-alpha.0 is used).
> 
> please note, that existing avatars are not rotated correctly after update, you can check that only with new avatars.
> 
> however, of course, may also be another issue :)

I just tried current master + #1837 and the problem still persists, it happens in my Samsung phone (stock android with android 10)

The issue is with the camera, selecting from gallery works fine for me, but taking pictures with the camera option, returns the image rotated, I can see it rotated in the "crop image" view before selecting

![Screenshot_20210315-054127_DeltaLab](https://user-images.githubusercontent.com/24558636/111133975-824f1c80-8551-11eb-88fb-2fe3f53cf931.jpg)

**Comment by Hocuri** [comment]
> The issue is with the camera

This made me think, maybe the bug is in the installed camera app (DC doesn't take the image but asks an installed camera app to do so)? Maybe you can try install another camera app (e.g. OpenCamera) and see if the issue persists?

**Comment by darhma** [comment]
@Hocuri I tried with open camera and the problem is still present.
@adbenitez the issue, for me, is present also selecting photos from gallery and I don't see the rotation in the "crop image" screen but only once done.

I am among those who initially reported the bug in version 1.15.0 beta (I've downloaded the nightly version on 10/03/2021). I have a xiaomi redmi note 7 with lineageos 17.1 without google services.

**Comment by havet** [comment]
The problem persists on my phone, a Sony with Android Androoid 8.0.0 and DC v. 1.14.5
When I choose a picture with "Pictures" it's shown alright, and the square crop is shown alright, but when I choose it it rotates.

**Comment by havet** [comment]
Unfortunately, this kind of problem might be a problem for the Delta Chat reputation: it appears simple to the user, who might infer that the software is of poor quality, containing lots of errors. The surface is important.

**Comment by adbenitez** [comment]
> Unfortunately, this kind of problem might be a problem for the Delta Chat reputation: it appears simple to the user, who might infer that the software is of poor quality, containing lots of errors. The surface is important.

yes, was onboarding some friend and after installing the app the first steps were setting an avatar picture and all the images she tried were rotated, was really annoying, had to do a trick (editing the image in delta chat then exporting the image to finally set it as avatar)

**Comment by Gh0stz0x** [comment]
@gerryfrancis  I try to ask if the bug still persists

**Comment by Gh0stz0x** [comment]
Looks like it's been fixed!
If it comes back I'll reopen it!
Thanks everyone!
To you too Gerry, who are here to help!

**Comment by gerryfrancis** [comment]
@Gh0stz0x Hey, many thanks for coming back and reporting here, glad to know that the issue is solved! :)
